drivers/volume/portworx/portworx.go

```diff
@@ -499,7 +508,7 @@ func (d *portworx) StopDriver(nodes []node.Node, force bool) error {
 	for _, n := range nodes {
 		logrus.Infof("Stopping volume driver on %s.", n.Name)
 		if force {
-			pxCrashCmd := "sudo kill -9 px-storage"
+			pxCrashCmd := "sudo pkill -9 px-storage"
```

Comment:
this should be pkill right? or either grep process ID of px-storage and then use kill -9?